manual/cypher/cypher-docs/src/docs/graphgists/intro/data-structures.adoc

```diff
@@ -86,17 +96,18 @@ For instance, you might want to further match patterns in the graph.
 Or you passed in a collection of values but now want to create or match a node or relationship for each element.
 Then you can use the `UNWIND` clause to unroll a list into a sequence of rows again.
 
-For instance, a query to find the top 5-co-actors and then follow their movies and again list the cast for each of those movies:
+For instance, a query to find the top 3 co-actors and then follow their movies and again list the cast for each of those movies:
 
 [source,cypher]
 ----
-MATCH (a:Person)-[:ACTED_IN]->(m:Movie)<-[:ACTED_IN]-(colleague:Person)
-WITH colleague, count(*) as frequency, collect(distinct m) as movies
+MATCH (actor:Person)-[:ACTED_IN]->(movie:Movie)<-[:ACTED_IN]-(colleague:Person)
+WHERE actor.name < colleague.name
```

Comment:
No, it's to make every pair appearing only once. Otherwise we'll get both A, B and B, A.
I bet there's a better way?
Note that I want to stay away from ordering by (node) id.